src/main/java/ru/mtsbank/fintech/animal_repository/AnimalRepositoryImpl.java

```diff
@@ -53,20 +55,21 @@ public Map<String, LocalDate> findLeapYearNames() {
         for (Map.Entry<String, List<AbstractAnimal>> entry : animalMap.entrySet()) {
             leapYearBirthAnimal.putAll(entry.getValue().stream().distinct()
                     .filter(value -> value.getBirthDate().isLeapYear())
-                    .collect(Collectors.toMap(AbstractAnimal::getName, AbstractAnimal::getBirthDate,(existingValue, newValue) -> newValue)));
+                    .collect(Collectors.toMap(AbstractAnimal::getName, AbstractAnimal::getBirthDate, (existingValue, newValue) -> newValue)));
         }
         return leapYearBirthAnimal;
     }
 
     /**
      * <b>findOlderAnimal</b>
      * возвращает Map животных, старше заданного возраста или самое взрослое животное
+     *
      * @param age искомый возраст
      * @return Map<AbstractAnimal, Integer> - ключ: животное, значение: возраст
      */
     @Override
     public Map<AbstractAnimal, Integer> findOlderAnimal(int age) {
-        if (age < 0) throw new IllegalArgumentException();
+        if (age < 0) throw new IllegalValueException("Incorrect Value !");
```

Comment:
Смотри, по типу исключения `IllegalValueException` и так предполатгается, что выброшено оно в связи с "Incorrect Value". 
Для чего пишется текст ошибки: чтобы понимать где и что некорректно.
лучше заменить на
```suggestion
        if (age < 0) throw new IllegalValueException("Incorrect age");
```